src/main/java/lotto/lotto/LottoSeller.java

```diff
@@ -0,0 +1,38 @@
+package lotto.lotto;
+
+import lotto.number.LottoNumbers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LottoSeller {
+    static final int PRICE_OF_A_TICKET = 1000;
+    private static final int ZERO = 0;
+
+    private LottoSeller() {}
+
+    public static Lotto buy(final int payment) {
+        validate(payment);
+
+        LottoTickets lottoTickets = LottoTickets.init(buyAutoLotto(payment));
+
+        return Lotto.init(payment, lottoTickets);
+    }
+
+    private static List<LottoTicket> buyAutoLotto(final int payment) {
```

Comment:
```suggestion
    private static List<LottoTicket> buyAutoLottoTickets(final int payment) {
```
Lotto라는 이름의 클래스가 존재해서, 명확히 구별하는게 좋을 것 같아요

Reply:
넵 수정해야겠네요